rust/frontend/src/server.rs

```diff
@@ -1224,3 +1576,51 @@ async fn v1_deprecation_notice() -> Response {
     );
     (StatusCode::GONE, Json(err_response)).into_response()
 }
+
+////////////////////////////////////////////////////////////
+/// OpenAPI
+////////////////////////////////////////////////////////////
+struct ChromaTokenSecurityAddon;
+impl Modify for ChromaTokenSecurityAddon {
+    fn modify(&self, openapi: &mut utoipa::openapi::OpenApi) {
+        let components = openapi.components.as_mut().unwrap();
```

Comment:
rust convention - please comment unwraps as follows 

`// NOTE(hammadb) - This unwrap is usually safe, and will crash the service on initialization if its not.`

Reply:
We should not unwrap unless we have to, and in this case I think its ok since its at boot time and will fail immediately right?

Also - you can use .expect("It should be able to get components as mutable") which makes this more debuggable if it ever fails

Reply:
Updated - thanks